app/main.py

```diff
@@ -1,3 +1,63 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
```

Comment:
It is recommended to add a docstring to the class and its methods to describe what they do. This makes your code more understandable for other developers.